### PR TITLE
Deploy dynflowd [deb]

### PR DESCRIPTION
**Pull request by ekohl**
Replaces https://github.com/theforeman/foreman-packaging/pull/1954 because this branch name should correctly build.

### CODE REVIEW

**Comment by ekohl** [comment]
[test]

File: debian/jessie/foreman/rules [hunk 1]
```diff
@@ -26,3 +26,9 @@ build:
 	/bin/sed -ri '1sXenv rubyXenv foreman-rubyX' bin/*
 	/bin/sed -ri 's~^BUNDLER_CMD=""~BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"~' script/foreman-rake
 	/bin/mkdir -p .ssh
+	# Ensure backwards compatiblity by having dynflowd aliased to foreman-tasks
+	/bin/mv /usr/sbin/dynflow-executor /usr/sbin/dynflowd
```
Comment (mmoll) [inline debian/jessie/foreman/rules]: I guess this should be handled in the rules file?

File: debian/jessie/foreman/rules [hunk 2]
```diff
@@ -26,3 +26,9 @@ build:
...
+	/bin/ln -s /usr/sbin/dynflowd /usr/sbin/ruby-foreman-tasks
```
Comment (mmoll) [inline debian/jessie/foreman/rules]: there's a links file, see foreman-proxy

**Comment by mmoll** [comment]
There won't be a trusty release fo 1.16, so these files can just get ommited.

**Comment by ekohl** [comment]
> There won't be a trusty release fo 1.16, so these files can just get ommited.

And Jessie has been dropped from nightly too. Should they still be included?

File: debian/jessie/foreman/rules (hunk 1 above)
Comment (ekohl) [reply debian/jessie/foreman/rules]: It also breaks the build:

```
17:16:10 /bin/mv /usr/sbin/dynflow-executor /usr/sbin/dynflowd
17:16:10 /bin/mv: cannot stat '/usr/sbin/dynflow-executor': No such file or directory
```

**Comment by mmoll** [comment]
> And Jessie has been dropped from nightly too. Should they still be included?

Yes, because this is making cherry-picking to 1.16 easy. ;)

**Comment by ekohl** [comment]
And should we still ship the ```dynflowd.init``` file or can we rely on systemd now?

**Comment by mmoll** [comment]
> And should we still ship the dynflowd.init file or can we rely on systemd now?

This is kind of disputed, I guess... Ppersonally I'd just 🚢 only systemd files, but at least jessie can also be ran without systemd. If it's cheap for us to just :shipit: the sysvinit files, why not?

**Comment by ekohl** [comment]
Fair enough.

**Comment by mmoll** [comment]
I think as the foreman package is cdbs based, the dh-stuff can't be used (but didn't check).

**Pull request closed without merging**